pkg/errors/errors.go

```diff
@@ -26,6 +26,10 @@ var ErrInvalidWorkflowType = errors.New("invalid workflow type, must be either '
 var ErrInvalidProcessType = errors.New("invalid process type, must be either 'trigger', 'task' or 'exit'")
 var ErrInvalidProcessObjectStoreScope = errors.New("invalid process object store scope, must be either 'product' or 'workflow'")
 var ErrInvalidNetworkingProtocol = errors.New("invalid networking protocol, must be either 'UDP' or 'TCP'")
+var ErrInvalidProcessCPU = errors.New("invalid process CPU, must be of form '1', '0.5' or '100m'")
```

Comment:
This should reference that the CPU value is for the limit somewhere.

Maybe change it to:
```suggestion
var ErrInvalidProcessCPULimits = errors.New("invalid process CPU limit value, must be of form '1', '0.5' or '100m'")
```

Reply:
This error refers to both the limit and request variables

Reply:
Yep, but both limit and requests are resource limits